src/App.tsx

```diff
@@ -50,22 +51,117 @@ import ContentManager from "./api/ContentManager"
 import { Plugins, AppState } from '@capacitor/core';
 import NotificationsAssistant from "./api/NotificationsAssistant"
 
+
+
 // init content manager
 let cm = new ContentManager();
 
 const App: React.FC = () => {
 
+
   // capacitor notifications setup
-  useIonViewDidEnter(() => {
+  useIonViewWillEnter(() => {
     // refresh notifications on app enter
-    Plugins.App.addListener("appStateChange", (state: AppState) => {
+   
+  });
+
+  useEffect( () => {
+
+    //console.log("Annyeonghaseyo")
+
+    let notificationAssistant = new NotificationsAssistant();
+
+    var remain:Boolean;
+    var starting:Number;
+    var ending:Number;
+
+    notificationAssistant.NoNotificationsRemaining().then((remaining) => {
+      if(!remaining){
+        console.log("There are notifications that are already scheduled");
+      }else{
+        console.log("There are notifications that aren't scheduled")
+      }
+      remain = remaining;
+    } ).then(()=>
+      notificationAssistant.GetStart()
+    ).then((start)=>{
+      starting = start
+    }).then(()=>notificationAssistant.GetEnd()).then((end)=>{
+      ending = end
+    }).then(()=>{
+
+      var dateToday = new Date()
+
+      var hour = dateToday.getHours()
+      var minutes = dateToday.getMinutes()
+
+      var militaryTime:Number = hour*100 + minutes
+
+      if(remain && (militaryTime < starting && militaryTime < ending)){
+        notificationAssistant.NotificationSetter();
```

Comment:
This is not good. This code should refill the notifications whatever time of the day it is called.

Currently, notifications will only refill if you open the app within the start and end time set for the notifications. This needs to change.